Development/cpprest/access_token_error.h

```diff
@@ -0,0 +1,67 @@
+#ifndef CPPREST_ACCESS_TOKEN_ERROR_H
+#define CPPREST_ACCESS_TOKEN_ERROR_H
+
+#include "nmos/string_enum.h"
+
+namespace web
+{
+    namespace http
+    {
+        namespace oauth2
+        {
+            // experimental extension, for BCP-003-02 Authorization
+            namespace experimental
+            {
+                // for redirect error
+                // "If the resource owner denies the access request or if the request
+                // fails for reasons other than a missing or invalid redirection URI,
+                // the authorization server informs the client by adding the following
+                //  parameters to the query component of the redirection URI using the
+                //  "application/x-www-form-urlencoded" format"
+                // see https://tools.ietf.org/html/rfc6749#section-4.1.2.1
+
+                // for diret error
```

Comment:
Should this be "For redirect error:" ?
```suggestion
                // For redirect error:
```

Reply:
should be `for direct error:`
```suggestion
                // for direct error:
```